client/my-sites/subscriptions/turn-off-emails.tsx

```diff
@@ -0,0 +1,23 @@
+import { useTranslate } from 'i18n-calypso';
```

Comment:
I am sorry I did not make this clear in the issue description but the idea is that we define all the components dedicated for our SubscriptionManager in the `@automattic/subscription-manager` package. The package is located at [`packages/subscription-manager`](https://github.com/Automattic/wp-calypso/tree/trunk/packages/subscription-manager).

I would suggest naming all the setting components with Setting postfix in their name (e.g.: `BlockEmailsSetting`, `EmailFormatSetting` etc.), and to define them in `packages/subscription-manager/src/components/UserSettings`.

Reply:
No, no, it was my mistake. I saw your previous PR, and I saw the package there. I just ignored it 🤣